src/js/components/modals/AppModalComponent.jsx

```diff
@@ -49,7 +49,8 @@ var AppModalComponent = React.createClass({
     return {
       fields: AppFormStore.fields,
       errorIndices: {},
-      responseErrorMessages: {}
+      responseErrorMessages: {},
+      force: false
```

Comment:
I don't like the fact that I had to add a new variable in the state, but it was the quickest thing to do. Do you see any other option?